hydrator-common/src/main/java/io/cdap/plugin/common/ReferenceBatchSink.java

```diff
@@ -41,4 +46,24 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
     IdUtils.validateReferenceName(config.referenceName, collector);
     collector.getOrThrowException();
   }
+
+  /**
+   * Record field-level lineage for batch sink plugins (WriteOperation). This method should be called from prepareRun of
+   * any batch sink plugin.
```

Comment:
nit - put a new line after this, the convention is to have a new line before the params